src/Microsoft.Azure.WebJobs.Host/TypeUtility.cs

```diff
@@ -116,5 +118,20 @@ public static bool IsAsyncVoid(MethodInfo methodInfo)
         {
             return IsAsync(methodInfo) && (methodInfo.ReturnType == typeof(void));
         }
+                
+        private static bool HasConverterWorker<TAttribute, TSrc, TDest>(IConverterManager converterManager)
```

Comment:
I've moved to extensions methods for IConverterManager. 

---
In reply to: [99681763](https://github.com/Azure/azure-webjobs-sdk/pull/1006#discussion_r99681763) [](ancestors = 99681763)